.chloggen/custom_receiverhelper.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: receiverhelper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Update receiverhelper StartXOp and EndXOp methods with otelcol_receiver_refused_x with otelcol_receiver_failed_x in order to be able to distinguish internal vs downstream errors
+
+# One or more tracking issues or pull requests related to the change
+issues: ["https://github.com/open-telemetry/opentelemetry-collector/issues/12207"]
```

This should just be the issue number: (sorry, it's not obvious from the comment)
```suggestion
issues: [12207]
```

---

Done